code/game/gamemodes/events/meteors.dm

```diff
@@ -268,7 +268,7 @@
 	z_original = z
 
 
-/obj/effect/meteor/Move()
+/obj/effect/meteor/Move(NewLoc, Dir = 0, step_x = 0, step_y = 0, var/glide_size_override = 0)
```

Comment:
Why set default values of arguments for all inherited `Move` procs? Shouldn't they utilize default value of parent when present for consistency?

Reply:
If you don't re-specify the default value of the parameter in the child, it'll be null inside that child (if not provided), while it'll be correct in the parent only. They're not inherited so yes they have to be re-specified.

![rect_screenshot_temp](https://user-images.githubusercontent.com/8107459/48211941-e27a9600-e37a-11e8-87e2-d91d24f8104e.png)